cmd/podman/save.go

```diff
@@ -77,6 +77,9 @@ func saveCmd(c *cli.Context) error {
 	if !c.Bool("quiet") {
 		writer = os.Stdout
 	}
+	if !c.IsSet("output") {
+		writer = os.Stderr
```

Comment:
Eeek - careful. I like Matt's suggestion but this might not be the best way to do it: think about this combination: podman save --quiet myimage >/dev/null

Reply:
Does Go have the ternary operator? If so, perhaps:

    if !c.bool("quiet") {
        writer = c.IsSet("output") ? os.Stdout : os.Stderr
    }

Reply:
Nope, no ternary. Nested if statements are the next best thing.

Reply:
right, I didn't think of this case. Fixed it using nested if, works now.

Reply:
Sounds good to me but I'm not seeing the new code yet. Missing push?

Reply:
Oh, for balance, would it make sense to `s/Stdout/Stderr/` in `cmd/podman/load.go:95` ?

Reply:
I would be in favor of doing a general pass over our progress bar code and changing to `Stderr`. Might want to do that in a separate PR though

Reply:
@mheon So should it be to Stderr throughout? regardless of whether the output is being redirected?

Reply:
@umohnani8 Yeah, this seems more like debug output, which usually goes to Stderr instead of Stdout

Reply:
As I said before, can happen in another PR... I'll throw an LGTM on this for now